autonomi/src/client/high_level/data/private.rs

```diff
@@ -37,7 +37,7 @@ impl Client {
             "Fetching private data from Data Map {:?}",
```

Comment:
If it's being surfaced to the user, can we consistently use "datamap" as one word?

I know this isn't actually one of your changes, but it looks unprofessional on the UI if we use terms inconsistently.